Workshop on EDA.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Hey, that's a good change!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/niwako/garbanzo/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>